bitnami/kubeapps/values.yaml

```diff
@@ -690,7 +690,7 @@ apprepository:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
-  ## @param apprepository.initialRepos [array] Initial chart repositories to configure
+  ## @param apprepository.initialRepos Initial chart repositories to configure
```

Comment:
In this case, you do need the modifier. The rule is:
- Is an object/array set to an empty object `{}/[]` by default? Don't use it
- Is an object/array and I want to omit the subfields in the README.md? Use it

Reply:
Ahhh, ok, got it. I just removed any [array] and [object] and, when no changes were being generated in the readme, I made the commit. But, ok, let's keep it.